wpeverest/user-registration
# Addon - Range field support

### All Submissions:






### Changes proposed in this Pull Request:
This PR provides support like icon in the form builder, form setting parse in the frontend registration form, and edit-profile page for range field of advanced fields addon.

### How to test the changes in this Pull Request:

1. Activate advanced fields addon and drag range field.
2. Set the settings of the range field as per your convenience and hit save.
3. Navigate to the frontend and edit-profile page and verify if all settings are reflected properly.

### Types of changes:




### Other information:




### Changelog entry

> Addon - Range field support
